### ISSUE TITLE
Why so pedantic?

### ISSUE THREAD

**Issue by jtojnar** [jtojnar/nixpkgs-hammering#2]
This [comment](https://github.com/NixOS/nixpkgs/pull/97563#issuecomment-690180172) nicely explains why we have such high standards for drive-by contributions. We should incorporate it into the README.

**Comment by blaggacao** [comment]
What frustrates me: there should be automation for all that is possible to automate.
The degree of automation still is relatively low (at least in my perception compared to other projects).

So thank you for this project! I hope it will make it into a bot, soon.

**Comment by davidak** [comment]
So basically:

> We have high standards for drive-by contributions because we have to maintain it afterwards.

It's actually the same as any other open source project.

**Comment by jtojnar** [comment]
Right, I have often seen people to not realize that or keep forgetting that.

**Comment by blaggacao** [comment]
Well, since we all enjoy philosophical discussions, let's not forget the all-embracing underlaying fundamental truth that contribution of any sort emanates out of a scarse resource itself.

The fact that there are usually more contributors than maintainers is a systemic skew which should not be misunderstood on the individual level. It is hard not to fall in this perception trap.

It should be always the optimal strategy to lower friction for **both** sides of this interaction.

The natural leadership here is upon the maintainers.
